### PR TITLE
chore: jumper clean up tracking tools

### PR DESCRIPTION
**Pull request by dennyscode**
Ticket: https://lifi.atlassian.net/browse/LF-7747

### CODE REVIEW

File: src/components/Menus/WalletMenu/WalletCard.tsx [hunk 1]
```diff
@@ -60,15 +63,19 @@ export const WalletCard = ({ account }: WalletCardProps) => {
...
-      trackPageload({
-        source: TrackingCategory.Wallet,
-        destination: 'blokchain-explorer',
-        url: account.blockChainExplorerUrl || '',
-        pageload: true,
-        disableTrackingTool: [EventTrackingTool.Cookie3],
+      trackEvent({
+        category: TrackingCategory.Pageload,
+        action: TrackingAction.PageLoad,
+        label: 'pageload-discord',
```
Comment (tcheee) [inline src/components/Menus/WalletMenu/WalletCard.tsx:71]: typo on the label no?

File: src/components/PoweredBy/PoweredBy.tsx [hunk 2]
```diff
@@ -40,18 +43,21 @@ export const PoweredBy = ({ styles }: PoweredByProps) => {
...
-    trackPageload({
-      source: TrackingCategory.PoweredBy,
-      destination: 'lifi-website',
-      url: lifiUrl,
-      pageload: true,
-      disableTrackingTool: [EventTrackingTool.ARCx, EventTrackingTool.Cookie3],
+    trackEvent({
+      category: TrackingCategory.Pageload,
+      action: TrackingAction.PageLoad,
+      label: 'pageload-discord',
```
Comment (tcheee) [inline src/components/PoweredBy/PoweredBy.tsx:49]: typo as well?

File: src/components/WelcomeScreen/WelcomeScreen.tsx [hunk 3]
```diff
@@ -64,14 +59,17 @@ export const WelcomeScreen = ({ closed }: WelcomeScreenProps) => {
...
-    trackPageload({
-      source: TrackingCategory.WelcomeScreen,
-      destination: 'docs-sc-audits',
-      url: auditsWelcomeUrl,
-      pageload: true,
-      disableTrackingTool: [EventTrackingTool.Cookie3],
+    trackEvent({
+      category: TrackingCategory.Pageload,
+      action: TrackingAction.PageLoad,
+      label: 'pageload-discord',
```
Comment (tcheee) [inline src/components/WelcomeScreen/WelcomeScreen.tsx:66]: same

File: src/types/userTracking.ts [hunk 4]
```diff
@@ -25,60 +17,14 @@ export interface TrackEventProps {
...
-  action: string;
-  category: string;
-  chain: ChainID;
-  value?: number;
-  disableTrackingTool?: EventTrackingTool[];
-  data: Record<string, unknown>;
-  txhash: string;
-}
-export interface TrackChainSwitchProps {
-  account?: Account;
```
Comment (tcheee) [inline src/types/userTracking.ts:38]: How do we track chainSwitch now?

File: tests/e2e.spec.ts [hunk 5]
```diff
@@ -70,7 +70,7 @@ test.describe('Jumper full e2e flow', () => {
     await expect(page.getByRole('menu')).toBeVisible();
     await itemInMenu(page, 'Jumper Profile');
     expect(await page.url()).toBe(profileUrl);
-    await page.locator('.profile-page').isVisible({timeout: 15000});
+    await page.locator('.profile-page').isVisible({ timeout: 15000 });
```
Comment (tcheee) [inline tests/e2e.spec.ts:73]: please merge last develop to avoid issue here

File: src/components/Menus/WalletMenu/WalletCard.tsx (hunk 1 above)
Comment (dennyscode) [reply src/components/Menus/WalletMenu/WalletCard.tsx:71]: Yesss! Good find, thanks!

File: src/components/PoweredBy/PoweredBy.tsx (hunk 2 above)
Comment (dennyscode) [reply src/components/PoweredBy/PoweredBy.tsx:49]: Yes, right. Good find!!!

File: src/types/userTracking.ts (hunk 4 above)
Comment (dennyscode) [reply src/types/userTracking.ts:38]: I have added it now to the Widget.tsx.

File: src/components/Widgets/Widget.tsx [hunk 6]
```diff
@@ -175,39 +190,35 @@ export function Widget({
...
-    integratorStringByType,
+    widgetTheme.config.appearance,
+    widgetTheme.config.theme,
+    multisigWidget,
     isMultisigSigner,
     multisigSdkConfig,
-    multisigWidget,
-    setWalletSelectMenuState,
-    starterVariant,
-    theme.breakpoints,
```
Comment (oktapodia) [inline src/components/Widgets/Widget.tsx:196]: deleting those might break the theme switching

**Review by oktapodia — CHANGES_REQUESTED** [review]
Mainly a question

**Review by admsk2 — APPROVED** [review]

**Review by oktapodia — APPROVED** [review]
LGTM

**Pull request merged**